### PR TITLE
Add network logs UI for vscode

### PR DESCRIPTION
**Pull request by pahans**
## Purpose
Add network logs UI for vscode

selection_010

### CODE REVIEW

File: composer/tracing/src/TraceList.tsx [hunk 1]
```diff
@@ -19,8 +19,9 @@
 import * as React from "react";
 import * as _ from 'lodash';
 import moment from 'moment';
-import { Grid, Icon } from 'semantic-ui-react';
-import './index.scss';
+import { Table, Icon } from 'semantic-ui-react';
+// import './index.scss';
```
Comment (jeradrutnam) [inline composer/tracing/src/TraceList.tsx]: Shall we remove the commented line?

File: composer/tracing/src/TraceLogs.tsx [hunk 2]
```diff
@@ -19,30 +19,54 @@
 import * as React from "react";
 import ToolBar from './ToolBar';
 import TraceList from './TraceList';
-import './index.scss';
+// import './index.scss';
```
Comment (jeradrutnam) [inline composer/tracing/src/TraceLogs.tsx]: Shall we remove the commented line ?

File: composer/tracing/src/TraceLogs.tsx [hunk 3]
```diff
@@ -19,30 +19,54 @@
...
             filteredTraces: props.traces,
         };
     }
+
+    onSelected(selected: any) {
+        if (selected.id === this.state.selected) {
+            this.setState({
+                selected: undefined
+            });
+            this.props.onSelected(undefined);
```
Comment (jeradrutnam) [inline composer/tracing/src/TraceLogs.tsx]: Can we use boolean value instead `undefined` ?

```suggestion
            this.props.onSelected(false);
```

Comment (pahans) [reply composer/tracing/src/TraceLogs.tsx]: selected is a `string` here

**Pull request closed without merging**